source/application/wintaser.cpp

```diff
@@ -4783,6 +4792,7 @@ int APIENTRY wWinMain(HINSTANCE hInstance,
 
 	Load_Config();
 
+    Utils::Exceptions::InitWindowsExceptionsHandler();

```

Comment:
I am delaying it slightly on purpose, since other DLLs may replace it, although I don't think that will happen with default Windows DLLs... I found nothing on google about that.